dpdispatcher/machines/pbs.py

```diff
@@ -324,11 +323,11 @@ def resources_subfields(cls) -> List[Argument]:
                 dict,
                 [
                     Argument(
-                        "sge_pe_name",
+                        "pe_name",
```

Comment:
For the backward compatibility, please add `alias = ["sge_pe_name"]`. Btw, it seems out of the scope of the PR title.

Reply:
I forget to include this in previous PR. 
can you show me where to put alias?

Reply:
Please check https://docs.deepmodeling.com/projects/dargs/en/latest/api/dargs.html#dargs.Argument